conans/client/configure_build_environment.py

```diff
@@ -186,7 +187,17 @@ def make(self, args=""):
         with environment_append(self.vars):
             str_args = args_to_string(args)
             cpu_count_option = ("-j%s" % cpu_count()) if "-j" not in str_args else ""
-            self._conanfile.run("make %s %s" % (str_args, cpu_count_option))
+            if self._conanfile.settings.get_safe("os") == "Windows":
+                if self._compiler == "Visual Studio":
+                    build_command = find_executable("jom.exe")
+                    if build_command:
+                        self._conanfile.run("%s %s %s" % (build_command, str_args, cpu_count_option))
+                    else:
+                        self._conanfile.run("nmake %s" % str_args)
+                else:
+                    self._conanfile.run("mingw32-make %s %s" % (str_args, cpu_count_option))
+            else:
```

Comment:
What about the case of msys2 with make in windows? It's a little tricky, why that specific order and no other?
Probably we need to add a parameter "make_command" to the `make` method to call explicitly to the indicated by the user.

Reply:
maybe the best it to use find_executable and use mingw32-make if present, otherwise make ?

Reply:
Yes, could be. And I probably would introduce the optional parameter to override the conan "choose" of the correct make, if needed